packages/core/database/src/migrations/users.ts

```diff
@@ -46,7 +48,21 @@ export const createUserMigrationProvider = (db: Database): UserMigrationProvider
 
   const umzugProvider = new Umzug({
     storage: createStorage({ db, tableName: 'strapi_migrations' }),
-    logger: console,
+    logger: {
+      info(message) {
```

Comment:
It's kind of annoying and error-prone to have to redefine the same logger again and again for each migration. Is there a way to factorize/DRY that a bit, or is it on purpose?

Reply:
it's the kind of stuff you don't want to factorize too early. the to info funcs are different and it's enough to duplicate the rest instead of making a factory with options honestly